### ISSUE TITLE
Unable to get access token

### ISSUE THREAD

**Issue by Aadityajoshi151** [swappyison/JiocinemaDownloader#7]
Hi
I am trying to run this, but I am not able to get access token.
There is no `key-jiocinema` present in the network tab in my case. There isn't any `apis-jiocinema.voot` URL also.
My environment:
**_Brave browser_** Version 1.57.62 Chromium: 116.0.5845.180 (Official Build) (64-bit)
Also tried with **_Google Chrome_** Version 116.0.5845.179 (Official Build) (64-bit)
I am trying to download a show that does not require login to watch. However, I tried by logging in also.
Please let me know how can I move forward with this.
Thank you.

**Comment by swappyison** [comment]
please post the video link if possible

**Comment by Aadityajoshi151** [comment]
Hi
I was able to get the access token today using the file `jioaccess.js`. I had to install `puppeteer` package via NPM to get it to work, but it worked after that.
With that working, I was unable to download initially because `N_m3u8DL-RE.exe` file was not working on my Linux environment, but it worked when I switched to a Windows environment.
I am finally able to download the video I wanted.
Closing the issue now.
Thanks.
